web-ui/src/main/resources/catalog/views/default/templates/recordView.html

```diff
@@ -334,11 +334,14 @@ <h2 data-translate="">metadataInformation</h2>
           </ul>
           </p>
           <!-- TODO: use map config -->
+          <img class="gn-img-extent"
+               data-ng-src="region.getmap.png?mapsrs=EPSG:3857&width=250&background=osm&geomsrs=EPSG:4326&geom={{mdView.current.record.getBoxAsPolygon()}}"/>
+      </section></br>
```

Comment:
You should drop those 3 lines - it makes the HTML not well formed and will fail on the image - see the changes line 342